src/foam/nanos/crunch/ui/ScrollSectionWizardView.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.crunch.ui',
+  name: 'ScrollSectionWizardView',
+  extends: 'foam.u2.View',
+  // extends: 'foam.u2.detail.MultipleModelSectionedDetailView',
+
+  documentation: `Simply displays "sections" consecutively.`,
+
+  imports: [
+    'notify',
+    'stack'
+  ],
+
+  implements: [
+    'foam.mlang.Expressions'
+  ],
+
+  messages: [
+    { name: 'SaveSuccess', message: 'Your progress has been saved.' },
```

Comment:
double check with others, but believe we are using constant style naming for the message keys. 
Indeed, this is the case just below.